### PR TITLE
fix: Badge airdrops in batches

### PR DESCRIPTION
**Pull request by ncomerci**

### CODE REVIEW

**Review by 1emu — APPROVED** [review]

File: src/back/jobs/BadgeAirdrop.ts [hunk 1]
```diff
@@ -28,5 +30,7 @@ async function runQueuedAirdropJobs() {
 }
 
 async function giveAndRevokeLandOwnerBadges() {
-  await BadgesService.giveAndRevokeLandOwnerBadges()
+  if (config.getEnv() === Env.PRODUCTION) {
```
Comment (andyesp) [inline src/back/jobs/BadgeAirdrop.ts:33]: `config` is frontend only 🤔

Comment (1emu) [reply src/back/jobs/BadgeAirdrop.ts:33]: can we use isProdEnv()?

Comment (andyesp) [reply src/back/jobs/BadgeAirdrop.ts:33]: Yes!

File: src/entities/Snapshot/utils.ts [hunk 2]
```diff
@@ -203,3 +204,8 @@ export function isSameAddress(userAddress?: string | null, address?: string | nu
     getChecksumAddress(userAddress) === getChecksumAddress(address)
   )
 }
+
+export async function getUsersWhoVoted(usersWithoutBadge: string[]) {
```
Comment (andyesp) [inline src/entities/Snapshot/utils.ts:208]: Maybe `getUsersFromVotes`? And rename `usersWithoutBadge` to `users`? Or move this to Badge utils if we want to make it specific to badges.

**Review by andyesp — APPROVED** [review]

**Pull request merged**